server/src/main/java/org/apache/iotdb/db/mpp/sql/planner/plan/node/write/InsertRowNode.java

```diff
@@ -223,7 +209,7 @@ public void transferType(List<MeasurementSchema> measurementSchemas)
       for (int i = 0; i < measurementSchemas.size(); i++) {
         if (measurementSchemas.get(i) == null) {
```

Comment:
what is equal to null mean?  add javadoc

Reply:
I'm also confused about this. @MrQuansy